tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -104,6 +104,33 @@ Feature: Module
     When I uninstall module "ps_featuredproducts" with deleteFile false
     Then I should have an exception that module is not installed
 
+  Scenario: Install module
+    When I install module "ps_featuredproducts"
```

Comment:
```suggestion
    Given module ps_featuredproducts has following infos:
      | technical_name | ps_featuredproducts |
      | version        | 1.0.0               |
      | enabled        | false                |
      | installed      | false               |
    When I install module "ps_featuredproducts"
```
This is a bit redundant because the module was uninstalled in the previous scenario, it means scenarios in this feature are co-dependent We try to avoid this to keep scenarios as independent as possible but I'd say in this case we can make an exception and it makes sense That said it doesn't hurt to emphasize what is the expected beginning status of the module before we use the install action